### PR TITLE
Added a warning about the email body in phishing layout (XSOAR)

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10249

## Description
Added the same warning we have in XSIAM, to the XSOAR phishing layout:

image

### CODE REVIEW

File: Packs/Phishing/ReleaseNotes/3_6_18.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Layouts
+
+##### Phishing Incident v3
+
+- Added a warning about possible presence of malicious links in the Email Body section in the layout.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_6_18.md:6]:
```suggestion
Added a warning about the possible presence of malicious links in the Email Body section in the layout.
```

**Comment by ShirleyDenkberg** [comment]
@idovandijk Doc review completed.

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json [hunk 2]
```diff
@@ -517,7 +516,7 @@
                         "y": 14
                     },
                     {
-                        "description": "",
+                        "description": ":warning: Links may be malicious!",
```
Comment (ssokolovich) [inline Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json:519]: Quesiton: is that presented by defualt? 
I mean we have an option to hide out the description so I hope we turned that on!

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json [hunk 3]
```diff
@@ -1357,6 +1356,8 @@
     "system": false,
     "version": -1,
     "fromVersion": "6.1.0",
-    "description": "",
-    "marketplaces": ["xsoar"]
+    "marketplaces": [
+        "xsoar"
```
Comment (ssokolovich) [inline Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json:1360]: we also have something for XSIAM right? should we add the same message there as well ?

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json (hunk 2 above)
Comment (ssokolovich) [reply Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json:519]:
```suggestion
                        "description": ":warning: This is the original HTML Email Section. Clicking on Links and URLs can be lead to a malicious site! ",
```

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json (hunk 3 above)
Comment (idovandijk) [reply Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json:1360]: In XSIAM we have the message as I added it in the past :D

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json (hunk 2 above)
Comment (idovandijk) [reply Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json]: The hiding option is for the section name, but the description will show

File: Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json (hunk 3 above)
Comment (ssokolovich) [reply Packs/Phishing/Layouts/layoutscontainer-Phishing_v_3.json:1360]: Cool - just align the text between the layouts of xsoar and xsiam :)

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**